src/utils/repo/commits.ts

```diff
@@ -0,0 +1,79 @@
+import * as Octokit from '@octokit/rest';
+import { ICommitWithFileChanges, Pick2, IChangedFile } from '../../types';
+
+/**
+ * For each commit in a list of commit IDs, determine which
+ * files were added, changed or removed
+ *
+ * @param github GitHub API
+ * @param owner name of repo owner
+ * @param repo name of repo
+ * @param commitList array of commit Ids
+ *
+ * @example
+ *
+ * const shas = [
+ *  '3551adb35febbc3299e47d970784e3fb3cccb912',
+ *  '105916e1b20780929c671e28a538dd1d165743e0'
+ * ];
+ *
+ * const commitFileChanges =
+ *   await getFileChangesForCommits(githubApi, 'mike-north', 'shipit-bot', shas);
+ *
+ * commitFileChanges[0].sha; // 3551adb35febbc3299e47d970784e3fb3cccb912
+ * commitFileChanges[0].files[0].filename;  // "abc.txt"
+ * commitFileChanges[0].files[0].status;    // "added"
+ * commitFileChanges[0].files[0].additions; // 10
+ * commitFileChanges[0].files[0].deletions; // 0
+ * commitFileChanges[0].files[0].changes;   // 0
+ */
+export async function getFileChangesForCommits(
```

Comment:
This seems kinda odd, it seems 100% duplicated from #17.

Reply:
yeah, it's literally the same commit.
```
branch 17 ------A-------
                |
                #17

branch 18 ------A-------B------ (created from branch 17)
                        |
                        #18
```